collector/project.go

```diff
@@ -106,6 +109,33 @@ func (p Project) Collect(c *CollectorOpts) interface{} {
 			p.Pod.Update(totalPo)
 			poUtils = append(poUtils, float64(totalPo))
 		}
+
+		// Apps
+		rancherLibraryName := ""
```

Comment:
This should be turned into a list of catalog names if we check all catalogs for the URL below and the logic should be moved outside the project for loop so it's not done for every project. The list only needs to be calculated once.